### ISSUE TITLE
add settings if default value for a field is given

### ISSUE THREAD

**Issue by rallek** [Guite/MostGenerator#887]
You have already this great settings for the images. Similar to this you may implement it for all fields where a default value is created within the model. 

I would prefere this instead of #263. 

This feature is not requested for 0.7.0

**Comment by Guite** [comment]
Wouldn't this result in a huge amount of setting fields... not sure whether this would be really useful...

**Comment by Portugao** [comment]
Sometimes this feature is useful. So it would be fine to have the option to activate it for each field; by default it could be inactive.

**Comment by Guite** [comment]
Note there is now an entity initialiser class (see #995) which allows to dynamically apply default values for newly created entities. This might be helpful for solving this issue, too.

**Comment by Guite** [comment]
Postponing, because this depends on #97.

**Comment by Guite** [comment]
As agreed at the camp we are going to utilise expressions to let the default value of an entity field point to / reference a variable field.

**Comment by Guite** [comment]
Closing in favour of #263
